### ISSUE TITLE
Data Updates Due for MSISDNs in Malaysia

### ISSUE THREAD

**Issue by atancasis** [DrHyde/perl-modules-Number-Phone#97]
Google's libphonenumber had [this](https://github.com/google/libphonenumber/commit/8e3bbe519b2bcf1e72df809624b4af6784928d41) recent update just a couple of days ago on May 6, wherein this [addendum](https://issuetracker.google.com/issues/151164629#comment4) was finally fixed [here](https://github.com/google/libphonenumber/commit/8e3bbe519b2bcf1e72df809624b4af6784928d41#diff-ddd69612678bd81dbd3fe3d26848fb7aR266) among (potentially) others.

To illustrate:

http://libphonenumber.appspot.com/phonenumberparser?number=6001172332796&country=MY

vs.

```perl
❯ perl -MNumber::Phone -E 'say $Number::Phone::VERSION; my $np = Number::Phone->new("6001172332796"); say $np->is_valid'
3.6003
Can't call method "is_valid" on an undefined value at -e line 1.
```

Fixing this might just require a rebuild of the library after pulling in the latest data. If an official release can be made following that, that would be much appreciated. Thanks!

**Comment by DrHyde** [comment]
Thanks. That should be +6011 in N::P, not 60011, but even so, I'll fix it to recognise the new range - you were right, it was just a case of re-running the build script. I'll push out a new release shortly.

I generally don't track all libphonenumber releases immediately, instead I do data updates every three months. The documentation on the update policy was not perhaps clear enough, so I've also updated that.

**Comment by DrHyde** [comment]
Fixed in 3.6004, which is currently on its way to the CPAN

**Comment by atancasis** [comment]
awesome, thank you SO MUCH @DrHyde, much appreciated!! 🙏
